nexus/src/app/sagas/snapshot_create.rs

```diff
@@ -181,6 +206,39 @@ impl NexusSaga for SagaSnapshotCreate {
             ACTION_GENERATE_ID.as_ref(),
         ));
 
+        builder.append(Node::action(
+            "destination_volume_id",
+            "GenerateDestinationVolumeId",
+            ACTION_GENERATE_ID.as_ref(),
+        ));
+
+        // At the beginning of this saga, save disk generation number. Compare
```

Comment:
nit: When this saga is created, do we have the disk db record available?  Could this go into the saga params instead?

Reply:
I removed this in 4ebe2979f36ed8706b474fe7c631eb721a2188c4